README.md

```diff
@@ -34,6 +33,121 @@ Special notes:
  * Since we cannot distribute Java Richards, a patch is required to derive the
    tracing version (`patches/trace_java_richards.diff`)
 
+## Using the scripts here to analyse an existing benchmark
+
+### Mandatory requirements
+
+  * Python 2.7 - the code here is not Python 3.x ready
+  * bzip2 / bunzip2 and bzip2 headers
+  * curl and headers
+  * gcc and make
+  * gfortran and headers
+  * liblzma library and headers
+  * Python modules: numpy, pip, libcap
+  * openssl and headers
+  * pkg-config
+  * pcre library and headers
+  * readline headers
+  * wget
+  * xorg headers
+
+### Optional requirements
+
+  * PyPy (will allow some code here to run faster)
+  * Python modules required for plotting: matplotlib, seaborn
+  * Required for generating LaTeX tables: a LaTeX distribution which provides
+    pdflatex, and the following packages: amsmath, amssymb, booktabs, calc,
+    geometry, mathtools, multicol, multirow, rotating, sparklines, xspace.
+    The TeX Live distribution should be fine.
+
+### Installing on Debian systems
+
+The following command will install all dependencies on Debian-based systems:
+
+```sh
+$ sudo apt-get install build-essential python2.7 pypy bzip2 libssl-dev \
+       pkg-config libcurl4-openssl-dev python-numpy python-matplotlib \
+       python-seaborn python-pip texlive-latex-extra wget xorg-dev \
+       libreadline-dev libbz2-dev liblzma-dev libpcre3-dev gfortran \
+
+
+$ ./build_stats.sh
+```
+
+### Setting up R
```

Comment:
Do we need this section, given that `build_stats.sh` does the necessary stuff? If so, we should say clearly "if you've used `build_stats.sh` you can skip this section."